app/views/browse/routes.html

```diff
@@ -7,7 +7,7 @@
       <div class="middle-header header-light">
         <div class="container-fluid">
           <div class="page-header page-header-bleed-right page-header-bleed-left">
-            <div class="pull-right" ng-if="'routes' | canI : 'create'">
+            <div class="pull-right" ng-if="!project || 'routes' | canI : 'create'">
```

Comment:
on the other case it was an ng-hide and it was on the project settings page, so if the project had not been loaded yet you were supposed to hide the actions.

here this is an ng-if so this should be `project && 'routes' | canI : 'create'`

@spadgett the issue is that the canI filter gets evaluated initially to false (the i havent loaded rules yet fallback) before the project has loaded, there is no other data bound at this level in the view so nothing will trigger another digest loop on this part of the view after the rules actually load unless you also make sure the project has loaded first.  Its not ideal.  On all of the individual resource pages the header is underneath an ng-if="whateverResource"  and that resource gets loaded after the project is loaded, so they dont have this problem...